tests/methodologies/para/test_ai_heuristic.py

```
@@ -1364,10 +1364,39 @@ def test_ensure_client_generic_exception(self, tmp_path: Path) -> None:
+        real_mode = f.stat().st_mode
+
+        now = time.time()
+
+        class MockStat:
+            st_mtime = now - (5 * 86400)
+            st_atime = now - (2 * 86400)
+            st_ctime = now - (5 * 86400)
+            st_birthtime = now - (5 * 86400)  # macOS — true birth time present
+            st_mode = real_mode
+
+        monkeypatch.setattr(Path, "stat", lambda _self, *args, **kwargs: MockStat())
```

Ruff (0.15.9)

[warning] Unused lambda argument: `args`

(ARG005)